libraries/lib-wave-track/WaveClip.h

```diff
@@ -155,18 +155,31 @@ class WAVE_TRACK_API WaveClip final :
    // Always gives non-negative answer, not more than sample sequence length
    // even if t0 really falls outside that range
    sampleCount TimeToSequenceSamples(double t) const;
-   sampleCount ToSequenceSamples(sampleCount s) const;
 
-   int GetRate() const { return mRate; }
+   int GetRate() const override
+   {
+      return mRate;
+   }
 
    // Set rate without resampling. This will change the length of the clip
    void SetRate(int rate);
 
-   double GetStretchRatio() const override { return 1.0; }
+   //! Stretches from left to the absolute time (if in expected range)
+   void StretchLeftTo(double to);
+   //! Sets from the right to the absolute time (if in expected range)
```

Comment:
Perhaps `StretchFromLeftTo`, `StretchFromRightTo` would be better names, making clearer which end is fixed and which one is moved.

Reply:
I think the comments don't match the function names and are confusing.

"StretchLeftTo" is really stretching the left bound while fixing the right bound.  The clip is stretched "From" the invariant right visible bound "to" a new left bound.

Or say "Stretches the left from its present position to the absolute time"

It would also be good to comment that nothing happens if `to` is right of the present right bound.

Mutatis mutandis for the other function.

Reply:
Indeed those are poor. Thanks for the suggestion, will address.